app/dashboard/tests/test_dashboard_models.py

```diff
@@ -490,11 +490,11 @@ def test_profile_activate_avatar():
             handle='fred',
             email='fred@localhost'
         )
-        CustomAvatar.objects.create(profile=profile, config="{}")
+        custom_avatar = CustomAvatar.objects.create(profile=profile, config="{}")
         social_avatar = SocialAvatar.objects.create(profile=profile)
         profile.activate_avatar(social_avatar.pk)
-        assert profile.avatar_baseavatar_related.get(pk=1).active is False
-        assert profile.avatar_baseavatar_related.get(pk=2).active is True
+        assert profile.avatar_baseavatar_related.get(pk=custom_avatar.id).active is False
+        assert profile.avatar_baseavatar_related.get(pk=social_avatar.id).active is True
```

Comment:
Since local dbs are reused and tests are executed in transactions that get rolled back, the ids may not match what was originally hard coded.